qiskit_experiments/hamiltonian_tomography/__init__.py

```diff
@@ -0,0 +1,37 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+=======================================================================================
+Hamiltonian Tomography Experiments (:mod:`qiskit_experiments.hamiltonian_tomography`)
+=======================================================================================
```

Comment:
I don't think this file is needed. I'd wait to have more experiments of the sort to avoid putting a single experiment in its own submodule. We can always change this file structure later on.